ipv-api/src/main/java/uk/gov/di/authentication/ipv/entity/SPOTClaims.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.authentication.ipv.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SPOTClaims {
+
+    @JsonProperty(value = "vot")
+    private String vot;
+
+    @JsonProperty(value = "vtm")
+    private String vtm;
+
+    public SPOTClaims(
```

Comment:
Not sure you need this constructor anymore ... if the `ObjectMapper` instance is instantiated with `ObjectMapperFactory.getInstance()` it is configured correctly to just work from the annotation on the private fields.

Reply:
As discussed, keeping the constructor but remove the JsonProperty annotations from it and then adding a default constructor which will be used for desearlizing in the tests.